starters/serverless-framework-sqs-dynamodb/package.json

```diff
@@ -0,0 +1,76 @@
+{
+	"name": "serverless-framework-sqs-dynamodb",
+	"version": "1.0.0",
+	"description": "A starter kit utilizing the Serverless Framework in conjunction with AWS DynamoDB & SQS.",
+	"author": "Dustin Goodman",
```

Comment:
Well I am THE author of this kit 😉 - I do want to start introducing authors to kits though for better attribution to make this project more open source friendly and crediting.

Reply:
@TapaiBalazs added some good context. I was under the impression that we didn't have it on the other kits, due to it being kits of us all. But maybe I'm just a bit too much of a communist 😉

Reply:
I just read more on the official docs on this: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#people-fields-author-contributors

Looks like we need author & contributor fields. IMO the person who defined the kit elements is the author and everyone who helped make it what it is are the contributors.

Reply:
I understand your point, but the generator script _currently_ does the following:

1. User runs the generator
2. User selects which kit they want, and then provide a name
3. the script downloads the folder of the starter kit from github
4. the package.json "name" and "version" properties get overwritten 

Therefore if Jane Doe generates this starter kit, their package.json will contain you as the author in their project. This is the reason why other kits does not have the author field in the package.json file.

Of course, the generator script can be updated to remove the author from the package.json, we could even add a third prompt, so the user could set their own author field during script run.

Reply:
Hmmm... this raises a good discussion point. Something I've been pondering is if we put a "starter-dev.json" or equivalent file in each kit that we use instead of the package.json for the CLI. In general, here are the problems I'm thinking about and wanting to solve:

1. Are contributors properly given credit for their submissions and work regardless of the initiation point?
2. Can we support more non-Node based starter kits? Right now the Deno kit has a bit of a hack to be included but what if we started to include non-JavaScript tooling as well? We need to maybe rethink our CLI strategy in this regard.

I'll be honest that I'm less concerned about the generated kit containing the original authors details. I think that's actually appropriate in this case. Renaming was to allow others to have a better project name custom to what they were doing out of the box. I think this other metadata file I'm thinking about is maybe the solve though so maybe this is a moot point.